src/cli/commands/snip.ts

```diff
@@ -19,15 +20,72 @@ interface SnipArgs extends MainArgs {
   destination: string;
   state?: string;
   ignore?: string | string[];
+  format?: string;
 }
 
-const handler = async ({
-  paths,
-  destination,
-  plugin,
-  state,
-  ignore,
-}: Arguments<SnipArgs>): Promise<void> => {
+export const doRst = async (
+  result: ParseResult
+): Promise<string | undefined> => {
+  const { source } = result;
+  if (
+    source.attributes["snippet"] === undefined ||
+    source.attributes["emphasize"] === undefined
+  ) {
+    return undefined;
+  }
+
+  // nasty hack to cover the suffixes/rst languages we use most often with Realm.
```

Comment:
Let's not bring Realm into this!
```suggestion
  // nasty hack to cover the suffixes/rst languages we use most often
```